package.json

```
@@ -29,11 +29,20 @@
     "lint:style": "stylelint '**/*.{,s}css' --ignore-path .gitignore --max-warnings 0",
     "lint:prettier": "prettier . --check --ignore-path .gitignore",
     "test": "jest --coverage",
+    "cy": "concurrently -k yarn:e2e:serve yarn:cypress:open",
+    "cypress:open": "cypress open",
+    "e2e": "concurrently -k --success first yarn:e2e:*",
+    "e2e:run": "wait-on http://localhost:6006 && cypress run -c video=true",
+    "e2e:serve": "NODE_ENV=e2e yarn start --ci --quiet",
+    "poste2e": "nyc report -r text -r text-summary",
+    "pree2e:run": "mkdir -p cypress",
```

I personally like reading scripts as pre, then script itself, then post, so it's more clear the sequence.

What do you think:

```suggestion
    "e2e": "concurrently -k --success first yarn:e2e:*",
    "poste2e": "nyc report -r text -r text-summary",
    "pree2e:run": "mkdir -p cypress",
    "e2e:run": "wait-on http://localhost:6006 && cypress run -c video=true",
    "e2e:serve": "NODE_ENV=e2e yarn start --ci --quiet",
```

---

Don't mind that at all. I never know when to use "pre/post" versus a simple `&&` either, I usually just avoid the script being too long.